src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Resources;
```

Comment:
Abhijeet, I looked into Nuget gallery. https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager/1.6.0-preview It seems like this version is the latest one...